### PR TITLE
fix #32: add optional task description to registerTasks()

### LINKED ISSUE #32: #registerTask(): Allow optional description as second param
[``grunt.task.registerTask`` allows for an optional second argument](http://gruntjs.com/creating-tasks#alias-tasks) which is used in the generated output of ``grunt -h``. Please allow submitting this second param, too.

### PR DESCRIPTION
**Pull request by MartinDelille**

### CODE REVIEW

File: test/registertask.js [hunk 1]
```diff
@@ -133,6 +175,33 @@ describe('#registerTask()', function () {
       });
     });
 
+    describe('a new group with description', function () {
+      it('a single task', function () {
+        gf = 'grunt.registerTask(\'deploy\', \'desc\', [\'foo\'])';
```
Comment (SBoudrias) [inline test/registertask.js]: `var gf`

**Comment by SBoudrias** [comment]
Hey, thanks for sending this PR.

I just made a summary overview and things looks mostly good. I'll review it correctly tonight or in the next few days.

File: test/registertask.js (hunk 1 above)
Comment (MartinDelille) [inline test/registertask.js]: I'm not sure what you mean by this? In the previous code, the *gf* variable wasn't local when reaffected.

Comment (SBoudrias) [inline test/registertask.js]: Oh you're right, that's how the code is currently. Fine by me then.

**Comment by SBoudrias** [comment]
Thanks for sending this PR! I'll make a new release right away.

**Pull request merged**